tests/installation.robot

```diff
@@ -1,26 +1,26 @@
 *** Settings ***
-Documentation    Test to verify installed operators screen for RHODA
-Metadata         Version    0.0.1
-Library         SeleniumLibrary
-Resource        ../resources/keywords/navigation.resource
-Resource        ../resources/keywords/operators.resource
-Suite Setup     Set Library Search Order  SeleniumLibrary
-Test Setup      Given The Browser Is On Openshift Home Screen
-Test Teardown   Close Browser
-Suite Teardown  Tear Down The Test Suite
+Documentation       Test to verify installed operators screen for RHODA
+Metadata            Version    0.0.1
+
+Library             SeleniumLibrary
+Resource            ../resources/keywords/navigation.resource
+Resource            ../resources/keywords/operators.resource
+
+Suite Setup         Set Library Search Order    SeleniumLibrary
+Suite Teardown      Tear Down The Test Suite
+Test Setup          Given The Browser Is On Openshift Home Screen
+Test Teardown       Close Browser
+
 
 *** Test Cases ***
 Scenario: Redhat Database Operator Installed On Operator Hub
-    [Tags]  smoke
+    [Tags]    smoke
     When User Navigates To OperatorHub Under Operators
     And User Searches Openshift Database Access Operator On RHODA Installed Cluster
     Then The Operator Tile Loads With Installed Status
 
 Scenario: Redhat Dbaas Operator Namespace Created For RHODA Installation
```

Comment:
I think the title here is incorrect. The scenario name should be:' Redhat Dbaas Operator Namespace **Filtered** For RHODA Installation' because 'redhat-dbaas-operator' already exists right?